app/models/movie.rb

```diff
@@ -0,0 +1,4 @@
+class Movie < ApplicationRecord
+    has_many :favorites
+         has_many :users, :through => :favorites
```

Comment:
The through syntax looks a little outdated. I think you'll want `through: :favorites`